lib/tasks/maintenance.rake

```diff
@@ -0,0 +1,23 @@
+namespace :maintenance do
+  task prison_uuids: :environment do
+    prison_uuid_mapping_path =
+      Rails.root.join('db', 'seeds', 'prison_uuid_mappings.yml')
+
+    prison_uuid_mappings = YAML.load(File.read(prison_uuid_mapping_path)) || {}
+
+    Dir[Rails.root.join('db', 'seeds', 'prisons', '*.yml')].each do |path|
+      filename = Pathname.new(path).basename.to_s
+      if prison_uuid_mappings[filename]
+        puts "Mapping exists for #{filename}. Skipping."
+      else
+        puts "No mapping for #{filename}. Generating."
+        prison_uuid_mappings[filename] = SecureRandom.uuid
+      end
+
+      YAML.dump(
+        prison_uuid_mappings,
+        File.open(prison_uuid_mapping_path, 'w')
```

Comment:
Well, it's complete lack of knowledge in this case. Mind you, even finding the documentation for the `YAML` module isn't trivial. ([It's actually under `Psych`](http://ruby-doc.org/stdlib-2.2.3/libdoc/psych/rdoc/Psych.html#method-c-dump) because … well … who knows?)